### DISCUSSION TITLE
Argo CD Operator v0.4.0 is out !!

### DISCUSSION THREAD

**Discussion by ishitasequeira** [argoproj-labs/argocd-operator#746] [Show and tell]
Argo CD Operator v0.4.0 is released to github . Thanks everyone who has contributed for the release. Cheers :argocd:
The Argo CD Operator v0.4.0 will be released to [operatorhub.io](http://operatorhub.io/) on Monday (August 1st).

Release Notes:
https://github.com/argoproj-labs/argocd-operator/releases/tag/v0.4.0

**Comment by ishitasequeira** [comment]
Argo CD Operator v0.4.0 is available on [operatorhub.io](http://operatorhub.io/) now

**Comment by iam-veeramalla** [comment]
Thanks @ishitasequeira
